Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -1016,4 +1018,25 @@ defineSuite([
             expect(color[3]).toEqual(255);
         });
     });
+
+    it('loads style from uri', function() {
+        // One building in each data set is always located in the center, so point the camera there
+        var center = Cartesian3.fromRadians(centerLongitude, centerLatitude, 5.0);
+        scene.camera.lookAt(center, new HeadingPitchRange(0.0, -1.57, 10.0));
+
+        return Cesium3DTilesTester.loadTileset(scene, withBatchTableUrl).then(function(tileset) {
+            // ${id} < 100 will always evaluate to true
+            var loadedStyle = new Cesium3DTileStyle(tileset, styleUrl);
+            return loadedStyle.readyPromise.then(function(style) {
```

Comment:
The user shouldn't have to wait for ready.  Instead, the style engine should check if the style is ready before applying it.  If the style is not ready and its apply is called, it should throw a `DeveloperError` (see Model.js for examples).

So this should be as simple as
```javascript
tileset.style = new Cesium3DTileStyle(tileset, styleUrl);
```
Then the style is basically asynchronously applied.